.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -231,6 +237,19 @@ jobs:
           path:
             artifacts.zip
       {%- endif %}
+      - name: Hold runner for 2 hours or until ssh sessions have drained
+        # Always hold for active ssh sessions
+        if: always()
+        run: |
+          echo "Holding runner for 2 hours until all ssh sessions have logged out"
```

Comment:
This means that even if the ssh session is on, the runner will still shut down after 2 hours regardless, right?

Reply:
Yes, this is consistent with CircleCI's current re-run with ssh functionality, see: https://circleci.com/docs/2.0/ssh-access-jobs/